server/lib/poller.js

```diff
@@ -65,12 +65,12 @@ class Poller
         }
 
         try {
-
             // Check accounts and operations!
-            let checkAccounts = await Config.findOrCreateDefaultBooleanValue(
+            let checkAccounts = false;
+            checkAccounts = await Config.findOrCreateDefaultBooleanValue(
                 'weboob-auto-merge-accounts'
             );
-
+        } finally {
```

Comment:
`try X/finally Y` is equivalent to `try X/catch{}; try Y/catch`. Can you do that instead? I've never been a big fan of finally...